### PR TITLE
implement orange logic search provider

### PR DESCRIPTION
**Pull request by petrjasek**

### CODE REVIEW

File: client/extensions/orangelogic-extension/package.json [hunk 1]
```diff
@@ -3,8 +3,8 @@
   "main": "src/extension.ts",
   "name": "orangelogic-extension",
   "scripts": {
-    "compile": "tsc",
-    "watch": "tsc -watch"
+    "compile": "",
```
Comment (tomaskikutis) [inline client/extensions/orangelogic-extension/package.json]: Why did you remove the script?

**Comment by petrjasek** [comment]
I think it's a runtime dependency given that we don't distribute js files. Question is if it needs to be compiled before importing it. But anyhow I think this should be defined in the extension, not in the code using it.

**Comment by tomaskikutis** [comment]
We do distribute `js` files. It's in the `dist` directory :smile: Yes it does need to be compiled before importing. It technically works even if we don't compile it, but it shouldn't and won't if we start distributing `superdesk-core` as a pre-bundled npm package which I think should be our goal(it will also help decouple the build process between core and client repos).
